content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -291,6 +290,12 @@ please see [here](https://kubernetes.io/docs/concepts/cluster-administration/net
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/v0.10.0/Documentation/kube-flannel.yml
 ```
+Note that `flannel` works on `amd64`, `arm`, `arm64` and `ppc64le`, but until version `0.11.0` is released
```

Comment:
not sure, the sentence is `, but until version 0.11.0 is released you need to use ...`.

Reply:
I was thinking of whether users are clear that the mentioned version `0.11.0` is the version of flannel. But guess we can assume that users know what the current version of K8S is and what the current version of flannel is. Just want to avoid the possible confusing :)

Reply:
ok, that makes sense! i will modify the sentence later today.